spec/views/admin/pages/index.html.slim_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
if you want to add some view spec, please write st. Otherwise, plz remove them, thanks!